src/common/components/layouts/pages/inspect/structure/Structure.js

```diff
@@ -1,5 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
+
```

Comment:
Fix the issue with jumping scroll. Scroll should stay the same if user selected an item that is visible in right pane with tree. See the video:
https://github-production-user-asset-6210df.s3.amazonaws.com/31737885/273913063-a10bd959-daf6-46f6-9929-825394163796.mp4